src/extension.ts

```diff
@@ -18,21 +18,29 @@ export function activate(context: vscode.ExtensionContext) {
         vscode.window.showInputBox({
             placeHolder: 'Enter the name of the new terminal'
         }).then(name => {
+            for (let j = 0; j <_terminals.length; j++) {
+                _terminals[j].hide();
```

Comment:
I think this may lead to a flicker from the panel hiding and then showing immediately. Maybe this should call `StatusBarTerminal.markHidden()` which is also called by `StatusBarTerminal .hide()`?